From 8769bba06b1632c6bd275f88c0fd10f591efdb9b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Krist=C3=BDna=20Streitov=C3=A1?= <kstreitova@suse.cz>
Date: Mon, 22 Sep 2014 10:41:14 +0200
Subject: [PATCH] fix manpage replace anacrontab with crontab

/etc/anacrontab replaced by /etc/crontab in the first occurance of it.
Fixes bnc#833240
---
 man/cron.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/cron.8 b/man/cron.8
index 633dbc0..9716f8f 100644
--- a/man/cron.8
+++ b/man/cron.8
@@ -53,7 +53,7 @@ for crontab files which are named after accounts in
 The found crontabs are loaded into the memory.
 .I Cron
 also searches for
-.I /etc/anacrontab
+.I /etc/crontab
 and any files in the
 .I /etc/cron.d
 directory, which have a different format (see
-- 
1.8.4.5

